external-crates/move/move-compiler/src/hlir/ast.rs

```diff
@@ -336,16 +336,16 @@ pub enum UnannotatedExp_ {
     Constant(ConstantName),
 
     ModuleCall(Box<ModuleCall>),
-    Builtin(Box<BuiltinFunction>, Box<Exp>),
+    Builtin(Box<BuiltinFunction>, Vec<Exp>),
```

Comment:
Thoughts on doing `Spanned<Vec<Exp>>` it might easier to massage the code in a few spots

Reply:
I think any errors that would have wanted that location have already been raised by this point (and it's retained for the `Multiple` form). Do you have something specific in mind?

Reply:
No, just that you are losing a `Loc` previously there. Not a big deal, just pointing it out